cmake/catkin_package.cmake

```diff
@@ -75,6 +77,7 @@
 #     LIBRARIES projlib1 projlib2
 #     CATKIN_DEPENDS roscpp
 #     DEPENDS Eigen
+#     DEFINES MYDEFINE=1
```

Comment:
I saw that. But internally they are stored as compile options. Why should the user not be able to pass compile options which are not definitions through `catkin_package`?